src/OnboardingSPA/components/ErrorState/Step/SiteGen/index.js

```diff
@@ -55,6 +56,10 @@ const SiteGenStepErrorState = () => {
 		setIsHeaderNavigationEnabled( true );
 		setDrawerActiveView( false );
 		setSidebarActiveView( false );
+		setDisableRetry(
+			currentData.sitegen?.siteGenErrorMeta?.retryCount >=
+				currentData.sitegen?.siteGenErrorMeta?.maxRetryCount
+		);
```

Comment:
The retryCount when sitemeta apis fail exceeds more than 3 so the button is already disabled in the error screen. 

<img width="2048" alt="Screenshot 2024-05-16 at 11 04 05 PM" src="https://github.com/newfold-labs/wp-module-onboarding/assets/4736718/9752ae94-00a5-4415-b757-73ec23234cdf">

Reply:
Yeah, I was thinking as every call costs us, would it be right to go forward considering all of them are failing.

Reply:
Fixed this please check, Thanks.